eng/testing/tests.mobile.targets

```
@@ -33,6 +36,16 @@
     <UseNativeHttpHandler>true</UseNativeHttpHandler>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetOS)' == 'iOS' and '$(UseDefaultiOSFeatureSwitches)' == 'true'">
```

this needs to check for iOS Simulator now that https://github.com/dotnet/runtime/pull/49305 is merged.

```suggestion
  <PropertyGroup Condition="('$(TargetOS)' == 'iOS' or '$(TargetOS)' == 'iOSSimulator') and '$(UseDefaultiOSFeatureSwitches)' == 'true'">
```

---

Thanks!